package.json

```diff
@@ -14,13 +14,20 @@
   "license": "MIT",
   "dependencies": {
     "body-parser": "1.17.2",
+    "cookie-parser": "^1.4.3",
```

Comment:
@Gwillison415 please remove this `^` as well as the others in the dependencies, thanks!